site/cypress/e2e/Cypress-Gradeable/rainbow_grading.spec.js

```diff
@@ -0,0 +1,63 @@
+describe('Test Rainbow Grading', () => {
+    beforeEach(() => {
+        cy.login('instructor');
+        cy.visit(['sample', 'config']);
+
+    });
+    it('Enable viewing of rainbow grades and generating the rainbow grading', () => {
+        cy.get('[data-testid="display-rainbow-grades-summary"]').then(($checkbox) => {
+            if (!$checkbox.prop('checked')) {
+                cy.get('[data-testid="display-rainbow-grades-summary"]').check();
+            }
+        });
+        cy.get('[data-testid="display-rainbow-grades-summary"]').should('be.checked');
+        cy.visit(['sample', 'reports', 'rainbow_grades_customization']);
+        cy.get('[data-testid="display-grade-summary"]').then(($checkbox) => {
```

Comment:
```suggestion
        cy.get('[data-testid="display-grade_summary"]').then(($checkbox) => {
```

This is one of the exception cases, as this is a procedurally generated testid, using the display value, that uses underscores.